jsonschema2pojo-cli/src/main/scripts/jsonschema2pojo.bat

```diff
@@ -15,4 +15,7 @@
 @REM
 
 @echo off
-java -jar "%~dp0/../lib/${project.build.finalName}.jar" %*
+
+pushd %~dp0

```

Comment:
What's the reason for this change? It looks like this will cause the script to always run in the working directory of the script itself, which doesn't seem correct.

Reply:
The manifest class path libs weren't resolving if I didn't do it.  If they provide a relative path as a parameter that may cause problems. Let me find a better solution.

Reply:
Remove all pushd and popd commands.